sched/Kconfig

```diff
@@ -894,22 +894,11 @@ config SCHED_INSTRUMENTATION
 	---help---
 		Enables instrumentation in scheduler to monitor system performance.
 		If enabled, then the board-specific logic must provide the following
-		functions (see include/sched.h):
+		functions (see include/sched_note.h):
 
-			void sched_note_start(FAR struct tcb_s *tcb);
-			void sched_note_stop(FAR struct tcb_s *tcb);
-			void sched_note_suspend(FAR struct tcb_s *tcb);
-			void sched_note_resume(FAR struct tcb_s *tcb);
+			void sched_note_add(FAR const void *note, size_t notelen);
 
-		If CONFIG_SMP is enabled, then these additional interfaces are
-		expected:
-
-			void sched_note_cpu_pause(FAR struct tcb_s *tcb, int cpu);
-			void sched_note_cpu_paused(FAR struct tcb_s *tcb);
-			void sched_note_cpu_resume(FAR struct tcb_s *tcb, int cpu);
-			void sched_note_cpu_resumed(FAR struct tcb_s *tcb);
-
-		NOTE: These are internal OS interfaces and are called at at very
+		NOTE: This is an internal OS interfaces and are called at at very
```

Comment:
```suggestion
		NOTE: This is an internal OS interface and is called at very
```

Reply:
@ghn-certi, I split the Kconfig change to another patch: please give your feedback here: https://github.com/apache/incubator-nuttx/pull/1751